File: c/1.c
#include <stdio.h>

int main()
{
  int sum = 0;
  int i = 0;
  for (i = 0; i < 1000; i++) {
    if (i % 3 == 0) {
      sum += i;
    } else if (i % 5 == 0) {
      sum += i;
    }
  }
  printf("%d\n", sum);
  return 0;
}

File: src/1.ts
import _ from 'lodash';

// Computes the sum of all the multiples of 3 or 5 below the given limit.
//
// See https://projecteuler.net/problem=1
export function compute(limit: number) {
  return _.range(limit)
    .filter(n => n % 3 === 0 || n % 5 === 0)
    .reduce((a, b) => a + b, 0);
}
